runestone/__main__.py

```diff
@@ -30,6 +30,8 @@ def init():
     conf_dict['author'] = click.prompt("Your Name ", default=getpass.getuser())
     conf_dict['project_title'] = click.prompt("Title for this project ", default="Runestone Default")
     conf_dict['python3'] = click.prompt("Use Simple Python3 Semantics ", default="false")
+    conf_dict['language'] = click.prompt("Locale ", default="en")
+    conf_dict['hide_control_captions'] = click.prompt("Hide control captions? ", type=bool, default=False)
```

Comment:
Hi and thanks for reply!

The hide_control_captions isn't directly related to the translation, but it is meant to be a global flag to hide captions of all controls in case they are unnecessary obtrusive, like they seem to be in our case. I mean, if a caption is not refered from the body text and doesn't provide any useful information to the reader, it's useless. It could be useful to create an instance-based flag for each control, too, but this is to be a global flag so that there wouldn't be the need to add instance-based flags everywhere throughout the produced material in case all the captions are to be hidden. Of course, only the ActiveCode control reacts to it for now via a simple if clause. I'm sorry I didn't put it into another pull request.

Aleksi